### ISSUE TITLE
Feature request - export table information by filtering on columns contained in / excluded from form(s)

### ISSUE THREAD

**Issue by ArneRustad** [MscrmTools/Javista.AttributesFactory#31]
Hi,

Once again a feature request from me. When exporting table information it would be very useful to be able to filter on whether a column is included in a specific form or alternatively excluded from a particular form. Especially the excluded from one (or more) form would be very useful in combination with the new delete column functionality. Then, we could easily export all columns of the table which are not part of our forms of interest and choose which we should delete from a much shorter list. The Metadata Document Generator implements this kind of filtering, therefore it should at least be possible, even though I do not know difficult it would be to implement.

Kind regards,
Arne

**Comment by ArneRustad** [comment]
I agree that adding it as a column to the exported Excel file would not be a good idea. What I was imagining was something similar to what Metadata Document Generator does, i.e. before starting the export, we can choose some filtering based on forms (does not affect the layout of the excel file). I include a screeenshot from the Metadata Document Generator below:

![image](https://user-images.githubusercontent.com/51076842/217020062-237503bd-f09a-4817-abc9-54a9dee6a97f.png)

![image](https://user-images.githubusercontent.com/51076842/217020190-43ea645f-d3ac-4fa1-ba95-55c25b2b8590.png)

In this way the exact same Excel format could be used (and the format would remain identical for export and import).

What do you think?

**Comment by MscrmTools** [comment]
Ok, I understand better.
This is something I can add

**Comment by ArneRustad** [comment]
Fantastic!
